repo/packages/H/hdfs/0/command.json

```diff
@@ -1,5 +1,5 @@
 {
   "pip": [
-    "git+https://github.com/mesosphere/dcos-hdfs.git@0.1.0#dcos-hdfs=0.1.0"
+    "git+https://github.com/mesosphere/dcos-hdfs.git@0.1.1#dcos-hdfs=0.1.1"
```

Comment:
but if you mean rewriting the hdfs universe package to take different config parameters, its not on the roadmap yet.